modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -14,11 +14,26 @@
 namespace cv {
 namespace gapi {
 
+/**
+ * The enum specified format of result that you get from @ref cv::gapi::stereo.
+ */
 enum class StereoOutputFormat {
-    DEPTH_FLOAT16,
-    DEPTH_FLOAT32,
-    DISPARITY_FIXED16_11_5,
-    DISPARITY_FIXED16_12_4
+    DEPTH_FLOAT16, ///< Floating point 16 bit value, CV_16FC1
+    DEPTH_FLOAT32, ///< Floating point 32 bit value, CV_32FC1
+    DISPARITY_FIXED16_11_5, ///< 16 bit signed: first bit for sign,
+                            ///< 10 bits for integer part,
+                            ///< 5 bits for fractional part.
+    DISPARITY_FIXED16_12_4, ///< 16 bit signed: first bit for sign,
+                            ///< 11 bits for integer part,
+                            ///< 4 bits for fractional part.
+    DEPTH_F16, ///< Floating point 16 bit value, CV_16FC1
```

Comment:
can this be defined as `= DEPTH_FLOAT16`? I believe we introduce some extra numeric values here - this is what we'd need to avoid.

Reply:
Also: I asked these the new names to unify with the OpenCV naming convention 

`F16` is not OpenCV-ish, `16F` is (see `CV_16F`).

Reply:
`DEPTH_F16` -> `DEPTH_16F = DEPTH_FLOAT16`